test/integration/controller_integration_test.go

```diff
@@ -136,7 +135,7 @@ var _ = Describe("Cluster controller", func() {
 		})
 
 		It("replace non seed node", func() {
-			const shortWait = 2 * time.Second
+			const shortWait = 30 * time.Second
```

Comment:
Maybe this should be globally available const

Reply:
fixed